app/assets/javascripts/govuk_publishing_components/lib/selection-buttons.js

```diff
@@ -0,0 +1,126 @@
+// migrated from govuk_frontend_toolkit
+
+// NOTICE
+// IF you are using GOV.UK Elements 3.0,0 or above then you do not need to use
```

Comment:
We don't have any projects still using GOV.UK Elements – hence this script is not needed or used across GOV.UK applications.